### PR TITLE
Update azure pipeline

### PR DESCRIPTION
**Pull request by xuzhg**

### CODE REVIEW

File: pipelines/azure_pipelines_nightly.yml [hunk 1]
```diff
@@ -40,6 +40,11 @@ stages:
       inputs:
         version: 5.x
```
Comment (ElizabethOkerio) [inline pipelines/azure_pipelines_nightly.yml:41]: should version be 6.x here?

File: pipelines/azure_pipelines.yml [hunk 2]
```diff
@@ -31,6 +31,11 @@ jobs:
     inputs:
       version: 5.x
```
Comment (ElizabethOkerio) [inline pipelines/azure_pipelines.yml:32]: 6.x?

**Review by ElizabethOkerio — APPROVED** [review]

File: pipelines/azure_pipelines.yml (hunk 2 above)
Comment (xuzhg) [reply pipelines/azure_pipelines.yml:32]: I added a new task to install 6.x. Later, I'd like to remove .net 5.

File: pipelines/azure_pipelines_nightly.yml (hunk 1 above)
Comment (xuzhg) [reply pipelines/azure_pipelines_nightly.yml:41]: I added a new task to install 6.x. Later, I'd like to remove .net 5.

**Review by gathogojr — APPROVED** [review]

**Pull request merged**